### PR TITLE
Docs: Add current postgresql content for ldapsearchfilter in pg_hba.conf

### PR DESCRIPTION
**Pull request by dyozie**

### CODE REVIEW

File: gpdb-doc/markdown/security-guide/topics/Authenticate.html.md [hunk 1]
```diff
@@ -254,13 +254,18 @@ ldapbindpasswd
 ldapsearchattribute
 :   Attribute to match against the user name in the search when doing search+bind authentication.
 
-Example:
+ldapsearchfilter
+:   Beginning with Greenplum 6.22, this attribute enables you to provide a search filter to use when doing search+bind authentication. Occurrences of `$username` will be replaced with the user name. This allows for more flexible search filters than `ldapsearchattribute`.
+
+When using search+bind mode, the search can be performed using a single attribute specified with `ldapsearchattribute`, or using a custom search filter specified with `ldapsearchfilter`. Specifying `ldapsearchattribute=foo` is equivalent to specifying `ldapsearchfilter="(foo=$username)"`. If neither option is specified the default is `ldapsearchattribute=uid`.
```
Comment (lisakowen) [inline gpdb-doc/markdown/security-guide/topics/Authenticate.html.md:260]: do we want to explicitly call out that the user can specify one of ldapsearchattribute and ldapsearchfilter, but not both?

**Review by lisakowen — APPROVED** [review]

File: gpdb-doc/markdown/security-guide/topics/Authenticate.html.md (hunk 1 above)
Comment (huiliang-liu) [reply gpdb-doc/markdown/security-guide/topics/Authenticate.html.md:260]: Yes, only one of them is available in the same time

**Comment by huiliang-liu** [comment]
LGTM

**Review by huiliang-liu — APPROVED** [review]

File: gpdb-doc/markdown/security-guide/topics/Authenticate.html.md (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/markdown/security-guide/topics/Authenticate.html.md:260]: Added in separate commit.  Thanks!

**Pull request merged**